### DISCUSSION TITLE
New install but no face recogition

### DISCUSSION THREAD

**Discussion by Pengouin** [photoprism/photoprism#3247] [Q&A]
Hello,

I am a new user. I installed PhotoPrism with docker-compose and nginx on a dedicated AlmaLinux 8 server. All config settings are default.

I added 200 photos (mostly of my daughter). I manually launched an indexing. Then I went to People > New. There I have a "No people found" (But I have a lot of faces when I click on "Show all new faces"). I try to wait (several hours). I try to manually define a person name by editing a photo. I also try to run "faces update" command. But still nothing show in new people.

My photos are good quality jpeg with a lot of series of the same portrait photos (almost the same photo ... should be very easy to match).

Can you please help me ?

**Comment by lastzero** [comment]
https://docs.photoprism.app/user-guide/organize/people/#why-doesnt-the-new-faces-page-show-all-faces

**Reply by lastzero** [reply]
https://docs.photoprism.app/known-issues/#asian-faces-and-children

**Reply by Pengouin** [reply]
Yep I already read those pages but can't find any solution.

So :
- face:new search give me almost all photos uploaded (normal as there is faces on all of them)
- "Assign Names to Faces From People" not possible as there is nothing showed 
- "Assign Names to Faces From Photos Edit dialog" done with 2 photos of one person (but no other photo get match and added)

- Requirements : should be ok : 2 Xeon cores with 8 GB or RAM but no GPU (vSphere cloud server)
- No Asian Faces but there are children
- No change to background worker

I have this message when I run faces update 

INFO[2023-03-04T22:38:31Z] faces: found 1 new cluster
INFO[2023-03-04T22:38:31Z] faces: skipped cluster ZZCNNKMZ3EAEPLYQZPAJQXAO4YDYKRCS, embedding not distinct enough

**Reply by Pengouin** [reply]
Oh and I forget. There are children but also some adults.
